build_from_config

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
+
+
+OPT=""
+VAL=""
+get_opt_and_val() {
+    IFS=":" read -ra TEMP <<< "${LINE}"
+    OPT="${TEMP[0]}"
+    VAL="${TEMP[1]}"
+}
+
+
+MIN=""
+MAX=""
+PLACEHOLDER=""
+COMMANDS=""
+get_min_and_max() {
+    IFS="," read -ra TEMP <<< "${VAL}"
+    MIN="${TEMP[0]/ /}"
+    MAX="${TEMP[1]/ /}"
+    if [ "${MIN}" != "" ]; then
+        COMMANDS="${COMMANDS} && [ ${PLACEHOLDER} -ge ${MIN} ]"
+    fi
+    if [ "${MAX}" != "" ]; then
+        COMMANDS="${COMMANDS} && [ ${PLACEHOLDER} -le ${MAX} ]"
+    fi
+}
+
+
+get_min_and_max_float() {
```

Comment:
Can you add a comment in front of every function that states what is does where the output is written to (in which variables)?